CONTRIBUTING.md

```diff
@@ -1,31 +1,42 @@
 # Contributing to Handsontable
 
-Your contributions to this project are very welcome. If you want to fix a bug or propose a new feature, you can open a new Pull Request but first make sure it follows these general rules:
+Your contribution to Handsontable's codebase is most welcome. To fix a bug or propose a new feature, open a new pull request (PR), targeted at the `develop` branch.
 
-1. Sign the [Contributor License Agreement](https://goo.gl/forms/yuutGuN0RjsikVpM2) to allow us to publish your changes.
-2. Make changes to the appropriate project:
-    - Handsontable is located in the `./handsontable` subdirectory.
-    - The framework-specific wrappers are located in the `./wrappers` subdirectory.
-3. Make your changes on a separate branch. This will speed up the merging process.
-4. Always make the target of your pull request the `develop` branch, not `master`.
-5. Do not edit files in the following directories:
+To speed up the process of merging your changes, follow these rules:
+
+1. Sign the [Contributor License Agreement](https://goo.gl/forms/yuutGuN0RjsikVpM2), to let us publish your changes.
+2. Make your changes on a separate branch. This will speed up the merging process.
+3. Always target your PR at the `develop` branch, not the `master` branch.
+4. Make changes to the right project:
+    - The main Handsontable project is located in the `./handsontable/` directory.
+    - Framework wrapper projects are located in the `./wrappers/` directory.
```

Comment:
WDYT about adding the info for docs contributors?

Reply:
Some of the other rules (e.g. tests, changelog entry) wouldn't apply to documentation PRs. Maybe add a separate section that redirects to the docs' README, to keep all documentation info in a single place?

d630836

Reply:
Sounds good 👍